book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```

Comment:
- [ ] Kindly add a method of adding a rental in book class to establish the `has many` association

```ruby
  def add_rental(person, date)
    Rental.new(date, self, person)
  end

```